subsys/testsuite/unittest.cmake

```diff
@@ -36,7 +36,13 @@ list(APPEND INCLUDE
 
 if(CMAKE_HOST_APPLE)
 else()
+
+if(M64_MODE)
+set (CMAKE_C_FLAGS "-m64") #deprecated on macOS
```

Comment:
is m64 really deprecated on macOS? You want to remove this comment...

Reply:
I copy-pasted it. Will remove it.

Reply:
Fixed it. Please see the latest PR.